src/main/java/org/folio/consortia/messaging/listener/ConsortiaEventListener.java

```diff
@@ -0,0 +1,50 @@
+package org.folio.consortia.messaging.listener;
+
+import static org.folio.consortia.utils.TenantContextUtils.getFolioExecutionContextCreatePrimaryAffiliationEvent;
+import static org.folio.consortia.utils.TenantContextUtils.runInFolioContext;
+
+import org.folio.consortia.service.UserAffiliationService;
+import org.folio.spring.FolioModuleMetadata;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.stereotype.Component;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+@Component
+@RequiredArgsConstructor
+public class ConsortiaEventListener {
+
+  public static final String USER_CREATED_LISTENER_ID = "user-created-listener-id";
+  public static final String USER_DELETED_LISTENER_ID = "user-deleted-listener-id";
+
+  private final UserAffiliationService userAffiliationService;
+  private final FolioModuleMetadata moduleMetadata;
+
+  @KafkaListener(
+    id = USER_CREATED_LISTENER_ID,
+    topicPattern = "#{folioKafkaProperties.listener['user-created'].topicPattern}",
+    concurrency = "1",
+    containerFactory = "kafkaListenerContainerFactory")
+  public void userCreatedListener(String data, MessageHeaders messageHeaders) {
+    log.info("Received USER_CREATED message: {}", data);
```

Comment:
We tried to exclude some personal data from USER_CREATED payload, but we still have firstname/lastname - lets not log this data, I know it would be harder for investigation issues, maybe we can log just that we recieved event with some message header and later log some data that is not sensitive

Reply:
removed